notebooks/236-stable-diffusion-v2/236-stable-diffusion-v2-text-to-image.ipynb

```diff
@@ -2,10 +2,8 @@
  "cells": [
```

Comment:
<blockquote>Quantizing the rest of <strong>the pipeline</strong> does not significantly improve inference performance but can lead to a substantial degradation of accuracy.</blockquote><p>extra space</p><p> </p><blockquote>For this model we apply quantization in hybrid mode which means that we quantize: (1) <strong>model weights</strong> and (2) activations of layers without weights.</blockquote><p>I think it is better to specify that we apply WC to MatMul and Embedding ops. Convolution ops have weights, but  PTQ should be applied for them.</p><blockquote>Collect <strong>operation </strong>with weights.</blockquote><p>operations</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Thanks, fixed</p>